src/www/webpack_base.mjs

```diff
@@ -42,25 +42,29 @@ export const baseConfig = {
 };
 
 export const browserConfig = {
-  entry: [path.resolve(__dirname, "./style.css")],
+  entry: [path.resolve(__dirname, './style.css')],
   output: {
-    path: path.resolve(__dirname, "../../www"),
-    filename: "main.js",
+    path: path.resolve(__dirname, '../../www'),
+    filename: 'main.js',
   },
   module: {
     rules: [
       {
-        test: /\.css?$/,
-        use: ["style-loader", "css-loader"],
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader'],
+      },
+      {
+        test: /\.(png|woff2)$/,
+        use: ['file-loader'],
```

Comment:
Let's make sure the files are actually being output and linked from the index html

Reply:
If it needs to be in the assets folder, restrict the pattern to that folder.

Reply:
Yeah, turns out it needs to be (for now) - adding